### PR TITLE
Algolia search fix

### PR DESCRIPTION
**Pull request by flornkm**
Reducing costs, getting rid of RPC and making it possible to search for displayName + descriptions.

### CODE REVIEW

**Comment by samuelstroschein** [comment]
Does this fix work with alglia category pages or is an interim hack? https://github.com/opral/monorepo/issues/1970#issuecomment-1877795624

**Comment by flornkm** [comment]
@samuelstroschein it doesn't work as the searchFilters have to be set **before** every request and they seem to not be compatible as async functions because it just works on every second search, not on the first one. I couldn't get it running yet. 

I think I have to remove the RPC and include algolia directly on the website so it is pre-rendered based on the route, not on the request.

**Comment by flornkm** [comment]
@NilsJacobsen please review it as I got rid of a component on the homepage (an old one and the component was the initial reason we used RPC).

**Review by NilsJacobsen — APPROVED** [review]
You are right we do not need it anymore. 👍

File: inlang/source-code/website/package.json [hunk 1]
```diff
@@ -16,10 +16,10 @@
 		"node": ">=18.0.0"
 	},
 	"dependencies": {
-		"@lix-js/server": "workspace:*",
 		"@isomorphic-git/cors-proxy": "^2.7.1",
 		"@lix-js/client": "workspace:*",
 		"@lix-js/fs": "workspace:*",
+		"@lix-js/server": "workspace:*",
```
Comment (samuelstroschein) [inline inlang/source-code/website/package.json:22]: @flornkm this can't be. The website can't depend on the server when the server depends on the website.

Comment (samuelstroschein) [reply inlang/source-code/website/package.json:22]: My mistake this is the lix server.

Comment (samuelstroschein) [reply inlang/source-code/website/package.json:22]: @opral/lix well well this is even not a change. the dependency just changed from line 19 to 22. good example that semantic diffing is also desired for code

**Pull request merged**